doc/attachments.md

```diff
@@ -0,0 +1,53 @@
+#Attachments
+
+You may attach a file from memory or by a file path.
+
+## From file path
+
+```php
+// Some how load the file to memory
+$binaryFile = '[Binary data]';
+
+$mg->message()->send('example.com', [
+  'from'    => 'bob@example.com', 
+  'to'      => 'sally@example.com', 
+  'subject' => 'Test file path attachments', 
+  'text'    => 'Test',
+  'attachment' => [
+    ['filePath'=>'/tmp/foo.jpg', 'filename'=>'test.jpg']
+  ]
+]);
+```
+## From memory
+
+```php
+// Some how load the file to memory
+$binaryFile = '[Binary data]';
+
+$mg->message()->send('example.com', [
+  'from'    => 'bob@example.com', 
+  'to'      => 'sally@example.com', 
+  'subject' => 'Test memory attachments', 
+  'text'    => 'Test',
+  'attachment' => [
+    ['fileContent'=>$binaryFile, 'filename'=>'test.jpg']
+  ]
+]);
+```
+
+## Inline attachments
+
+```php
+// Some how load the file to memory
+$binaryFile = '[Binary data]';
```

Comment:
I don't think we need this here since this example is showing the `'filePath'` method.